api/src/test/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimControllerTest.java

```diff
@@ -155,19 +149,7 @@ void shouldReturnInternalServerErrorStatusWhenEligibilityStatusIsError() {
         assertThat(response).isNotNull();
         assertThat(response.getStatusCode()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR);
         assertThat(response.getBody()).isEqualTo(aClaimResultDTOWithClaimStatusAndNoVoucherEntitlement(ClaimStatus.ERROR));
-        verifyCreateClaimCalledCorrectly(claimant, dto);
-        verify(claimantConverter).convert(aValidClaimantDTO());
-    }
-
-    private void verifyCreateClaimCalledCorrectly(Claimant claimant, NewClaimDTO dto) {
```

Comment:
I'm not sure why you removed this method, it seems like a perfectly valid thing to do to make sure that the ClaimService was called correctly, I think this should be added back into these tests.

Reply:
We're no longer constructing a Claimrequest object in the controller, and this method was testing that the ClaimRequest was constructed correctly.

Reply:
Let me rephrase that then, you are no longer verifying that the claim service is being called by not using this method, so that should be reinstated.

Reply:
You've missed my point, we are no longer verifying that the ClaimService is being called with a `verify()` call on that mock, we should be verifying that.

Reply:
added verify on create claim